Workers no longer start, error in pubsub

As of this morning, when I gcloud app deploy, the workers no longer start.

I'm using Google AppEngine, Python Flexible Environment

[2017-08-24 19:41:31 +0000] [9] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/env/lib/python3.5/site-packages/gunicorn/arbiter.py", line 578, in spawn_worker
    worker.init_process()
  File "/env/lib/python3.5/site-packages/gunicorn/workers/base.py", line 126, in init_process
    self.load_wsgi()
  File "/env/lib/python3.5/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/env/lib/python3.5/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/env/lib/python3.5/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/env/lib/python3.5/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/env/lib/python3.5/site-packages/gunicorn/util.py", line 352, in import_app
    __import__(module)
  File "/home/vmagent/app/main.py", line 2, in <module>
    import episteme.factory
  File "/home/vmagent/app/episteme/factory.py", line 25, in <module>
    import episteme.model
  File "/home/vmagent/app/episteme/model/__init__.py", line 662, in <module>
    from . import user
  File "/home/vmagent/app/episteme/model/user.py", line 563, in <module>
    from episteme.mail import EpistemeMailManager
  File "/home/vmagent/app/episteme/mail/__init__.py", line 16, in <module>
    from google.cloud import pubsub
  File "/env/lib/python3.5/site-packages/google/cloud/pubsub.py", line 17, in <module>
    from google.cloud.pubsub_v1 import PublisherClient
  File "/env/lib/python3.5/site-packages/google/cloud/pubsub_v1/__init__.py", line 18, in <module>
    from google.cloud.pubsub_v1.publisher import Client as PublisherClient
  File "/env/lib/python3.5/site-packages/google/cloud/pubsub_v1/publisher/__init__.py", line 17, in <module>
    from google.cloud.pubsub_v1.publisher.client import Client
  File "/env/lib/python3.5/site-packages/google/cloud/pubsub_v1/publisher/client.py", line 27, in <module>
    from google.cloud.pubsub_v1.publisher.batch import thread
  File "/env/lib/python3.5/site-packages/google/cloud/pubsub_v1/publisher/batch/thread.py", line 22, in <module>
    from google.cloud.pubsub_v1.publisher import exceptions
  File "/env/lib/python3.5/site-packages/google/cloud/pubsub_v1/publisher/exceptions.py", line 19, in <module>
    from google.api.core.exceptions import GoogleAPICallError
ImportError: No module named 'google.api.core'

--- a/pubsub/setup.py
+++ b/pubsub/setup.py
@@ -51,6 +51,7 @@ SETUP_BASE = {
 
 
 REQUIREMENTS = [
+    'google-cloud-core >= 0.27.0, < 0.28dev',
     'google-gax >= 0.15.13, < 0.16dev',
     'googleapis-common-protos[grpc] >= 1.5.2, < 2.0dev',
     'grpc-google-iam-v1 >= 0.11.1, < 0.12dev',
@@ -60,7 +61,7 @@ REQUIREMENTS = [
 
 setup(
     name='google-cloud-pubsub',
-    version='0.28.0',
+    version='0.28.1',
     description='Python Client for Google Cloud Pub/Sub',
     long_description=README,
     namespace_packages=[

